working/0698 - Enhanced Default Constructors/proposal.md

```diff
@@ -41,126 +42,253 @@ Since a mixin application class cannot contain static members, there is no risk
 
 ### Goal
 
-We combine the forwarding constructor feature with default constructors and automatic initialization of declared instance variables. This *unifies* default constructors and mixin application forwarding constructors *and* adds more power to the default constructor for simple classes.
+We want to allow classes with fields that can be constructed without having to write constructors. That is, we want *initializing default constructors* which takes parameters class fields (with some exceptions) and initializes the fields.
```

Comment:
Some grammar errors here. Maybe "which takes parameters for each class's fields and initializes the fields from them."

Reply:
Rewritten.